### DISCUSSION TITLE
Installation Changes

### DISCUSSION THREAD

**Discussion by DamianNiolet** [AdityaGarg8/mbp-16.x-ubuntu-kernel#63] [Q&A] [answered]
Hello, all.  Don't know who can answer this, but I've noticed an issue when trying to triple-boot.  I can get Windows on no problem, but when it comes to adjusting the windows partition to make room for Ubuntu, I hit a road block.  In macOS, the plus sign is greyed out in diskutility.  No way to adjust the partitions there.  I used to be able to adjust the partitions during installation, but with the latest UbuntuT2 iso, that ability to change the size of the Windows partition (the numbers in the popup window) is gone.  I even tried a live Ubuntu, installing gParted, and the adjustments error out.  That's a separate issue though.  Would like to just get back to the way I used to be able to do it.  So, the question is, was the ability to change the Windows partition's size removed from the Ubuntu installation file?

**Comment by AdityaGarg8** [comment, marked as answer]
You can resize a windows partition using the windows disk management only.

Now what do you want to further do to the created partition? If you wanna merge, then you have to ensure that the partition is not behind the Ubuntu installation. You can share `diskutil list` from macOS to check the same.

Also, there is a possibility that the MBR has changed to hybrid from protected during installation. This also needs to be fixed. For this boot into Ubuntu, and in the terminal there run

```
sudo gdisk /dev/nvme0n1
```

If MBR in output is hybrid and not protected, press `x` and return for expert mode, then `n` to change to protected MBR and `w` to write changes to disk.

**Comment by DamianNiolet** [comment]
OK.  That works.  And I guess it's easier that way.

Yes, quite familiar with conversion from hybrid to protective after much research and trial and error.

Thanks for the help!
